test/user/test_user.py

```diff
@@ -82,6 +83,14 @@ def test_delete_user(self, user_collection_mock):
             User("test@nonexistent.com"),
         )
 
+    @patch("app.rewards.service.firestore_db")
+    def test_create_user(self, rewards_db_mock, user_collection_mock):
```

Comment:
Where is `user_collection_mock` being hooked up? I don't see a patch for it anywhere. Genuinely curious.

Reply:
The patch is at the class level, not the method level